KDICCore/KDICPlayer.swift

```diff
@@ -3,29 +3,40 @@ import AVFoundation
 
 private let streamURL = URL(string: "http://kdic.grinnell.edu/stream")!
 
+let kdicInstance = KDICPlayer();
+
+private var isPlaying = false;
+
 open class KDICPlayer {
   private static var player: AVPlayer = {
     let asset = AVURLAsset(url: streamURL)
     let playerItem = AVPlayerItem(asset: asset)
+    // 
     return AVPlayer(playerItem: playerItem)
   }()
   
-  open static var isPlaying = player.rate != 0
   
   open class func play() {
     player.play()
+    isPlaying = true
   }
   
   open class func pause() {
     player.pause()
+    isPlaying = false
   }
   
+    /** to fix **/
   open class func toggle() {
-    if isPlaying {
+    if isPlaying == true {
       pause()
     } else {
       play()
     }
   }
+    
+    open func currentlyPlaying() -> Bool {
```

Comment:
Use a semi-public property above to make a private setter with a public getter. I put some code above. Please then remove this function.